lib/core/utils/performance-timer.js

```diff
@@ -90,11 +90,12 @@ const performanceTimer = (() => {
       ) {
         // only output measures that were started after axe started, otherwise
         // we get measures made by the page before axe ran (which is confusing)
-        const axeStart =
-          window.performance.getEntriesByName('mark_axe_start')[0];
+        const { startTime } =
+          window.performance.getEntriesByName('mark_axe_start')[0] ||
+          window.performance.getEntriesByName('mark_audit_start')[0];
```

Comment:
This error is kinda making me wonder if we should rethink not having tests for this. WDYT?